GLOSSARY.md

```diff
@@ -18,12 +19,20 @@ Suggestion on words and terms:
 | functional component | componente funcional |
 | key | chave |
 | library | biblioteca |
-| lowercase | minúscula(s) |
+| lowercase | minúscula(s) / caixa baixa |
 | package | pacote |
 | React element | Elemento React |
 | React fragment | Fragmento React |
 | render | renderizar (verb), renderizado (noun)
 | uncontrolled component | componente não controlado |
+| uppercase | maiúscula(s) / caixa alta |
+| wrap | colocar dentro de / envolver com  |
```

Comment:
```suggestion
| to wrap | embrulhar |
```

Colocar no infinitivo, porque funciona como verbo. 
Exemplo: 'to wrap a component' -> 'embrulhar um componente'

Reply:
No exemplo dado: `to wrap a component` eu prefiro traduzir como `encapsular um componente` ao invés de envolver/embrulhar, o que acha @fjoshuajr?

Reply:
Boa, bem melhor. Era essa palavra que eu queria

Reply:
Ahahaha, tem algumas palavras que são difíceis de aparecer né? A gente fica com outra automaticamente na cabeça.

Tb prefiro encapsular.

Embrulhar faz parecer que vc vai dar o componente de presente!

Commitando.